<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fractal Library</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSlider" name="verticalSlider">
      <property name="maximumSize">
       <size>
        <width>14</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>14</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QScrollBar" name="scrollTimeSeries">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>13</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>34</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Вариант разбора №</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbTreeNo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>из</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTreeCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Анализ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>616</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Показывать эталонное дерево</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Глубина генерируемого дерева:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbGeneratorDeep">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTSSize">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="yMultSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionSave_time_series"/>
    <addaction name="action_LoadEnvironment"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_templates"/>
   </widget>
   <widget class="QMenu" name="menu_Analyse">
    <property name="title">
     <string>&amp;Analyse</string>
    </property>
    <widget class="QMenu" name="menuAnalyser_type">
     <property name="title">
      <string>Analyser type</string>
     </property>
     <addaction name="actionIdeal"/>
     <addaction name="actionSimple"/>
    </widget>
    <addaction name="actionBuild_tree"/>
    <addaction name="actionMarkup_TS"/>
    <addaction name="actionNext_fractal_level"/>
    <addaction name="menuAnalyser_type"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionCreate_TS"/>
    <addaction name="actionUndo_TS"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Analyse"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="actionLoad_templates"/>
   <addaction name="separator"/>
   <addaction name="actionBuild_tree"/>
   <addaction name="actionMarkup_TS"/>
   <addaction name="actionNext_fractal_level"/>
   <addaction name="separator"/>
   <addaction name="actionDebugWindow"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate"/>
   <addaction name="actionGenerete_one"/>
   <addaction name="actionStart_Generator"/>
   <addaction name="actionStop_Generator"/>
   <addaction name="actionClear_Time_Series"/>
   <addaction name="separator"/>
   <addaction name="actionGeneric_Test"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_LoadEnvironment">
   <property name="text">
    <string>&amp;Load environment</string>
   </property>
   <property name="toolTip">
    <string>Open environment setting file</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Step">
   <property name="text">
    <string>&amp;Step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open CSV</string>
   </property>
   <property name="toolTip">
    <string>Open CSV file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMarkup_TS">
   <property name="text">
    <string>Markup TS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionNext_fractal_level">
   <property name="text">
    <string>Next fractal level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad_templates">
   <property name="text">
    <string>Load templates file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCreate_TS">
   <property name="text">
    <string>Create TS</string>
   </property>
  </action>
  <action name="actionUndo_TS">
   <property name="text">
    <string>Undo TS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionDebugWindow">
   <property name="text">
    <string>Debug Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionTemplates_editor">
   <property name="text">
    <string>Templates editor</string>
   </property>
  </action>
  <action name="actionBuild_tree">
   <property name="text">
    <string>Build tree</string>
   </property>
   <property name="toolTip">
    <string>Markup time series and find all fractal levels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_time_series">
   <property name="text">
    <string>Save CSV</string>
   </property>
   <property name="toolTip">
    <string>Save time series to CSV file</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionStart_Generator">
   <property name="text">
    <string>Start Generator</string>
   </property>
  </action>
  <action name="actionStop_Generator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Generator</string>
   </property>
  </action>
  <action name="actionGenerete_one">
   <property name="text">
    <string>Generete one</string>
   </property>
  </action>
  <action name="actionClear_Time_Series">
   <property name="text">
    <string>Clear Time Series</string>
   </property>
  </action>
  <action name="actionGeneric_Test">
   <property name="text">
    <string>Generic Test</string>
   </property>
  </action>
  <action name="actionIdeal">
   <property name="text">
    <string>Ideal</string>
   </property>
  </action>
  <action name="actionSimple">
   <property name="text">
    <string>Simple</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
